rviz_common/test/config_test.cpp

```diff
@@ -71,6 +71,13 @@ TEST(Config, set_get_empty_value) {
   EXPECT_EQ("", s);
 }
 
+TEST(Config, mapGetValue_key_not_found_null_check) {
+  rviz_common::Config c;
+  QString s;
+  EXPECT_FALSE(c.mapGetString("non_existent_key", &s));
+  EXPECT_EQ(s, "");
```

Comment:
Should we also test that whatever value is in `QString s` is kept if the key doesn't exist?

i.e.:

```suggestion
  QString s;
  EXPECT_FALSE(c.mapGetString("non_existent_key", &s));
  EXPECT_EQ(s, "");
  QString s_default("my_default_value");
  EXPECT_FALSE(c.mapGetString("non_existent_key", &s));
  EXPECT_EQ(s, "my_default_value");
```